filebeat/harvester/harvester.go

```diff
@@ -0,0 +1,9 @@
+package harvester
+
+import uuid "github.com/satori/go.uuid"
+
+type Harvester interface {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type Harvester should have comment or be unexported